samples/zigbee/light_switch/src/main.c

```diff
@@ -429,7 +430,7 @@ void main(void)
 #if defined BUTTON_SLEEPY
 	if (dk_get_buttons() & BUTTON_SLEEPY) {
 		zigbee_configure_sleepy_behavior(true);
-		zigbee_power_down_unused_ram();
+		power_down_unused_ram();
```

Comment:
Should this call be safeguarded with `if (IS_ENABLED(CONFIG_RAM_POWER_DOWN_LIBRARY))` ?

Reply:
Fixed.